ansible/configs/ocs4-external-implementation/sample_vars/osp_vars.yml

```diff
@@ -25,8 +25,9 @@ platform: labs
 purpose: development
 
 # OpenShift Installer version to use
-#ocp4_installer_version: 4.5.3
-#ocp4_installer_version: 4.6.1
+# ocp4_installer_version: 4.5.3
```

Comment:
Why include this commented variable?

Reply:
I kept the commented variables as it is as it was in original config. Should I remove that ?